Forum/Forum Define Table.sql

```diff
@@ -0,0 +1,66 @@
+USE master
```

Comment:
Can you divide it into separates scripts in folder structure for future changes or adding new. 
You can add 2 folders Forum DDL and Forum DML
DDL ->Data Definition Language for example when you create new table
DML-> Data Modification Language when you insert static records or update existing ones for example the user Roles table